apps/innovations/v1-innovation-units-suggestions/index.ts

```diff
@@ -0,0 +1,70 @@
+import { mapOpenApi3 as openApi } from '@aaronpowell/azure-functions-nodejs-openapi';
+import type { AzureFunction, HttpRequest } from '@azure/functions';
+
+import { JwtDecoder } from '@innovations/shared/decorators';
+import type { AuthorizationService } from '@innovations/shared/services';
+import SHARED_SYMBOLS from '@innovations/shared/services/symbols';
+import type { CustomContextType } from '@innovations/shared/types';
+
+import type { ResponseDTO } from './transformation.dtos';
+
+import { container } from '../_config';
+
+import type { InnovationSupportsService } from '../_services/innovation-supports.service';
+import { JoiHelper, ResponseHelper, SwaggerHelper } from '@innovations/shared/helpers';
+import { ParamsSchema, ParamsType } from './validation.schemas';
+import SYMBOLS from '../_services/symbols';
+
+class V1GetInnovationQASuggestions {
+  @JwtDecoder()
+  static async httpTrigger(context: CustomContextType, request: HttpRequest): Promise<void> {
+    const authorizationService = container.get<AuthorizationService>(SHARED_SYMBOLS.AuthorizationService);
+    const innovationSupportsService = container.get<InnovationSupportsService>(SYMBOLS.InnovationSupportsService);
+
+    try {
+      const params = JoiHelper.Validate<ParamsType>(ParamsSchema, request.params);
+
+      const auth = await authorizationService
+        .validate(context)
+        .setInnovation(params.innovationId)
+        .checkAccessorType()
```

Comment:
If you want this just to be accessed by the QAs you can add that to the validation:

```suggestion
        .checkAccessorType({ organisationRole: [ServiceRoleEnum.QUALIFYING_ACCESSOR] })
```